src/Microsoft.Health.Fhir.CosmosDb.Core/Features/Storage/StoreProcedures/StoredProcedureMetadataBase.cs

```diff
@@ -12,20 +12,20 @@
 using Microsoft.Azure.Cosmos.Scripts;
 using Microsoft.Health.Core.Extensions;
 
-namespace Microsoft.Health.Fhir.CosmosDb.Features.Storage.StoredProcedures
+namespace Microsoft.Health.Fhir.CosmosDb.Core.Features.Storage.StoredProcedures
 {
-    public abstract class StoredProcedureBase : IStoredProcedure
+    public class StoredProcedureMetadataBase : IStoredProcedureMetadata
```

Comment:
Should we mark this class as abstract? 
In my mind this class will no be instantiated, unless by the inheriting classes.

Reply:
done